### PR TITLE
task4597 new post

### PR DESCRIPTION
**Pull request by laam55**
https://edu-redmine.framgia.vn/issues/4597

### CODE REVIEW

File: app/Http/Requests/WritePostRequest.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'title' => 'required|min:6|max:150|unique:posts,title',
+            'content' => 'required|max:5000',
+            'tags' => [
```
Comment (framgiaciv3) [inline app/Http/Requests/WritePostRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.KeySpecified</br>**Severity**: error</br>**File**: app/Http/Requests/WritePostRequest.php</br>**Line**: 32</br>Key specified for array entry; first entry has no key

File: app/Repositories/Eloquents/PostRepository.php [hunk 2]
```diff
@@ -266,4 +266,12 @@ public function voteBestAnswer($input)
             return false;
         }
     }
+    // insert records posts_tags table
+    public function createPostsTags($tags, $post_id)
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/PostRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostRepository.php</br>**Line**: 270</br>Variable "post_id" is not in valid camel caps format

File: app/Repositories/Eloquents/PostRepository.php [hunk 3]
```diff
@@ -266,4 +266,12 @@ public function voteBestAnswer($input)
...
+    {
+        $post = $this->model()->find($post_id);
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/PostRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostRepository.php</br>**Line**: 272</br>Variable "post_id" is not in valid camel caps format

**Comment by laam55** [comment]
ready

**Pull request merged**